src/Tizen.NUI/src/internal/WebView.cs

```diff
@@ -561,6 +561,24 @@ public void LoadUrl(string url)
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Hidden API (Inhouse API).
+        /// Loads a html.
+        /// Using Uri class to provide safe sevice and secure API.
+        /// </summary>
+        /// <param name="url">The path of Web.</param>
+        /// <exception cref="ArgumentNullException">Thrown when url is null.</exception>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void LoadUrl(Uri url)
```

Comment:
I think the url should be changed to uri.

Reply:
yes. your are right.
fixed.